packages/guui/components/Byline/Byline.tsx

```diff
@@ -126,10 +126,13 @@ export const Byline: React.FC<{
         {author.twitterHandle && (
             <div className={twitterHandle}>
                 <TwitterIcon />
```

Comment:
will this icon be read? I guess we don't want it to be read as the link is pulling all the work

Reply:
VoiceOver ignores it. I haven't tested this on any other screenreader as I'm WFH